maint/code-cleanup.sh

```diff
@@ -131,18 +132,17 @@ if [ "$got_file" = "1" -a "$all" = "1" ]; then
 fi
 
 if [ "$recursive" = "1" ]; then
-    for i in `git ls-files | egrep '(\.c$|\.h$|\.c\.in$|\.h\.in$|\.cpp$|\.cpp.in$)' | \
-	egrep -v "($ignore_list)"` ; do
-	${debug} indent_code $i
+    for i in `git ls-files | egrep "($filetype_list)" | egrep -v "($ignore_list)"` ; do
 	${debug} indent_code $i
     done
 elif [ "$all" = "1" ]; then
-    for i in `git ls-files | cut -d/ -f1 | uniq | egrep '(\.c$|\.h$|\.c\.in$|\.h\.in$|\.cpp$|\.cpp.in$)' | \
-	egrep -v "($ignore_list)"` ; do
-	${debug} indent_code $i
+    for i in `git ls-files | cut -d/ -f1 | uniq | egrep "($filetype_list)" | egrep -v "($ignore_list)"` ; do
 	${debug} indent_code $i
     done
 else
-    ${debug} indent_code $@
-    ${debug} indent_code $@
+    for i in $@; do :; done
+    filename=`echo $i | egrep "($filetype_list)" | egrep -v "($ignore_list)"`
```

Comment:
Are you trying to only take the last filename?

Reply:
There can only be one file name, but if you pass `—debug`, that shows up here too.